matrix-vis.js

```diff
@@ -43,7 +43,7 @@ function initialSync() {
             var room = data.rooms[i];
             if (room.room_id === roomId) {
                 // add new events to graph
-                console.log("Adding "+room.messages.chunk.length+" new events to the graph");
+            console.log("Adding "+room.messages.chunk.length+" new events to the graph");
```

Comment:
We **MUST** keep the indent. Instead, put the string concatenation on new lines.